spec/acceptance/odoo_spec.rb

```diff
@@ -5,8 +5,8 @@
 def odoo_supported_versions
   {
     'Debian' => {
-      '11' => ['14.0', '15.0', '16.0', 'system'],
-      '12' => ['14.0', '15.0', '16.0', '17.0'],
+      '11' => ['14.0', '15.0', '16.0', '17.0', '18.0', 'system'],
```

Comment:
This must be synced with `manifest/init.pp`.

```suggestion
      '11' => ['14.0', '15.0', '16.0', 'system'],
```